src/ol_infrastructure/applications/mitlearn/__main__.py

```diff
@@ -1329,60 +1010,49 @@
 }
 
 # Combine two var sources above with values explictly defined in pulumi configuration
-heroku_vars.update(**heroku_interpolated_vars)
-heroku_vars.update(**heroku_app_config.get_object("vars"))
+env_vars.update(**interpolated_vars)
+env_vars.update(**mitlearn_config.get_object("vars"))
 
 # Making these `get_secret_*()` calls children of the seemigly un-related vault mount `secret-mitopen/` tricks
```

Comment:
As long is it's not a huge amount of work, we should replace the `secret-mitopen` mount with an equivalent `secret-mitlearn` mount